docs/migrate-to-antrea.md

```diff
@@ -0,0 +1,63 @@
+# Migrate from other CNI to Antrea
+
+This document describes how to migrate from other CNIs to Antrea.
+
+NOTE: The following is a reference list of CNIs and versions we have already
+verified the migration process. CNIs and versions that are not listed here
+might also work, please create an issue if you run into problems during the
+migration to Antrea. In addition, during the migration process, no Kubernetes
+resources should be created or deleted, otherwise the migration process might
+fail or some unexpected problems might occur.
+
+| CNI     | Version |
+|---------|---------|
+| Calico  | v3.26   |
+| Flannel | v0.22.0 |
+
+The migration process is divided into three steps:
+
+1. Clean up old CNI.
+2. Install Antrea in the cluster.
+3. Restart all Pods in the cluster in-place.
+
+## Clean up old CNI
+
+Before installing Antrea in the cluster, the old CNI needs to be cleaned up. The
+cleanup process is different for different CNIs, but the basic idea is to
+remove the old CNI DaemonSet, binaries and configuration files from the cluster.
+If the old CNI use iptables to manage the network, the iptables rules need to be
+cleaned up as well.
+
+## Install Antrea
+
+The first step is to install Antrea in the cluster. You can follow the
+[installation guide](https://github.com/antrea-io/antrea/blob/main/docs/getting-started.md)
+to install  Antrea in the cluster. The following is an example of
+installing Antrea with `build/yamls/antrea.yml`:
+
+```bash
+$ kubectl apply -f build/yamls/antrea.yml --kubeconfig ~/.kube/config
+```
+
+## Restart Pods
+
+After Antrea is installed in the cluster, the next step is to restart all Pods
+in the cluster in-place. The reason for restarting all Pods is that Antrea
+needs to take over the network management and IPAM from the old CNI. In order
+to avoid the Pods being rescheduled, we restart all Pods in-place by deploying
+a DaemonSet named `antrea-migrator`, which will run a Pod on each Node. The
+`antrea-migrator` Pod will stop all containerd tasks of Pods with CNI network on
+each Node, and the containerd tasks will be restarted by the containerd
+service. In this way, all Pods in the cluster will not be rescheduled but will
+be restarted in-place.
```

Comment:
After reading the document, as an user I still don't know how I should run the migrator.
The point of the document should be to guide users to do it, then explain how it's achieved if they are interested.

Reply:
Add a command to run deploy the migrator.  But since the v1.15.0 version is not released and we do not recommend the users to use the dev version, I'm not sure whether this example is ok.